security/s2a/internal/crypter/counter_test.go

```diff
@@ -0,0 +1,91 @@
+package crypter
+
+import (
+	"math"
+	"testing"
+)
+
+type counterTest struct {
+	desc          string
+	counter, want uint64
```

Comment:
Can we change `want` to `expectedCounter`?

Reply:
Done